packages/components/src/internal/components/domainproperties/assay/AssayPropertiesPanel.tsx

```diff
@@ -101,60 +103,73 @@ const AssayPropertiesForm: FC<AssayPropertiesFormProps> = memo(props => {
                 </Row>
             )}
 
-            <Col xs={12} lg={appPropertiesOnly ? 12 : 6}>
+            <Col xs={12} lg={hideAdvancedProperties ? 12 : 6}>
                 <div className="domain-field-padding-bottom">
                     <SectionHeading title="Basic Properties" />
-                    <NameInput model={model} onChange={onInputChange} appPropertiesOnly={appPropertiesOnly} canRename={canRename} />
-                    <DescriptionInput model={model} onChange={onInputChange} appPropertiesOnly={appPropertiesOnly} />
+                    <NameInput
+                        model={model}
+                        onChange={onInputChange}
+                        hideAdvancedProperties={hideAdvancedProperties}
+                        canRename={canRename}
+                    />
+                    <DescriptionInput
+                        model={model}
+                        onChange={onInputChange}
+                        hideAdvancedProperties={hideAdvancedProperties}
+                    />
                     {model.allowPlateTemplateSelection() && (
                         <PlateTemplatesInput
                             model={model}
                             onChange={onInputChange}
-                            appPropertiesOnly={appPropertiesOnly}
+                            hideAdvancedProperties={hideAdvancedProperties}
                         />
                     )}
                     {model.allowDetectionMethodSelection() && (
                         <DetectionMethodsInput
                             model={model}
                             onChange={onInputChange}
-                            appPropertiesOnly={appPropertiesOnly}
+                            hideAdvancedProperties={hideAdvancedProperties}
                         />
                     )}
                     {model.allowMetadataInputFormatSelection() && (
                         <MetadataInputFormatsInput
                             model={model}
                             onChange={onInputChange}
-                            appPropertiesOnly={appPropertiesOnly}
+                            hideAdvancedProperties={hideAdvancedProperties}
                         />
                     )}
-                    {!appPropertiesOnly && model.allowQCStates && isAssayQCEnabled(moduleContext) && (
+                    {!hideAdvancedProperties && model.allowQCStates && isAssayQCEnabled(moduleContext) && (
                         <QCStatesInput model={model} onChange={onInputChange} />
                     )}
-                    {!appPropertiesOnly && model.allowPlateMetadata && (
+                    {(!appPropertiesOnly || isPlatesEnabled(moduleContext)) && model.allowPlateMetadata && (
```

Comment:
Note: this is the line with the true fix for this issue. The rest was renaming the usages of appPropertiesOnly vs hideAdvancedProperties to be better reflect reality of our product features.